keys/constants.go

```diff
@@ -164,6 +163,9 @@ var (
 	// TableDataPrefix prefixes all table data. It is specifically chosen to
 	// occur after the range of common user data prefixes so that tests which use
 	// those prefixes will not see table data.
+	//
+	// TODO(peter): Replace with TableDataMin and TableDataMax which are the min
```

Comment:
Yeah. That isn't actually a problem for table IDs which use `EncodeUvarint`, but safest to adjust the definition of `intMin`. As mentioned elsewhere, I started working on this TODO and ran into variety of test failures which I haven't dug to the bottom of yet.